### PR TITLE
GFORMS-1126 Added id to addGroup and deleteGroup links

### PR DESCRIPTION
**Pull request by girishkamat**

### CODE REVIEW

File: app/uk/gov/hmrc/gform/views/form/snippets/group.scala.html [hunk 1]
```diff
@@ -56,12 +56,10 @@
 
   @snippets
 
-  @formAction(prefix: String, mongoIdentifier: String) = @{ prefix + "-" + mongoIdentifier }
-
   @canAddAnother.fold(HtmlFormat.empty){ modelComponentId =>
     @govukButton(Button(content = Text(groupField.repeatAddAnotherText.map(ls => ls.value).getOrElse(messages("linkText.addRepeatingGroup"))),
         inputType=Some("submit"),
-        attributes = Map("formaction" -> uk.gov.hmrc.gform.gform.routes.FormController.updateFormData(formTemplateId, maybeAccessCode, sectionNumber, FastForward.Yes, AddGroup(modelComponentId)).path),
+        attributes = Map("id" -> "addRepeatingGroup", "formaction" -> uk.gov.hmrc.gform.gform.routes.FormController.updateFormData(formTemplateId, maybeAccessCode, sectionNumber, FastForward.Yes, AddGroup(modelComponentId)).path),
```
Comment (SandipKhedekar) [inline app/uk/gov/hmrc/gform/views/form/snippets/group.scala.html:62]: Static **id** could cause accessibility issue like duplicate ids.

**Review by SandipKhedekar — CHANGES_REQUESTED** [review]

File: app/uk/gov/hmrc/gform/views/form/snippets/group.scala.html (hunk 1 above)
Comment (girishkamat) [reply app/uk/gov/hmrc/gform/views/form/snippets/group.scala.html:62]: It was initially a dynamic id, just like delete. However I think tests use a single group in pages, so its no a concern . FYI, thats how it was before as well.

Comment (SandipKhedekar) [reply app/uk/gov/hmrc/gform/views/form/snippets/group.scala.html:62]: I think, you haven't pushed your local changes yet.

Comment (girishkamat) [reply app/uk/gov/hmrc/gform/views/form/snippets/group.scala.html:62]: Its already there, this is how Ratheesh wanted it. I have told him about your concerns, he will get back to me

**Pull request merged**